Whiskey/Tasks/Invoke-WhiskeyNUnit3Task.ps1

```diff
@@ -16,18 +16,10 @@ function Invoke-WhiskeyNUnit3Task
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    $nunitPackages = @('NUnit.ConsoleRunner')
-    if ($TaskParameter['SupportNUnit2'])
-    {
-        # NUnit.Console includes NUnit.ConsoleRunner and the NUnit2 Compatibility Extension(s)
-        # with a version mirroring that of the contained console runner. However the extra packages are
-        # not unnecessary in strict NUnit3-native test suites. Since the Console package includes ConsoleRunner
-        # it can be download first and the subsequent package installs will be idempotent.
-        # By installing the NUnit.ConsoleRunner package separately (and last) the final NUnit
-        # executable path can be readily determined.
-        # NUnit3 only
-        $nunitPackages = @('NUnit.Console', 'NUnit.ConsoleRunner')
-    }
+    # NUnit.Console pulls in ConsoleRunner (of the same version) as a dependency and several NUnit2 compatibility/extension packages.
+    # The ConsoleRunner packages is installed explicitly to resolve the tool/bin path from installed package location.
+    $nunitSupportPackage = 'NUnit.Console'
+    $nunitPackage = 'NUnit.ConsoleRunner'
 
     # Due to a bug in NuGet we can't search for and install packages with wildcards (e.g. 3.*), so we're hardcoding a version for now. See Resolve-WhiskeyNuGetPackageVersion for more details.
     $nunitVersion = '3.7.0'

```

Comment:
Maybe we should upgrade to latest NUnit 3 while we're at it? 3.12. I didn't realize in my earlier review we were pinned to a specific version.

Reply:
Updated to 3.10 (latest version of https://www.nuget.org/packages/NUnit.ConsoleRunner/ ; core library versioning is close-yet-independent.)